pallets/runtime/src/bridge.rs

```diff
@@ -253,6 +314,61 @@ decl_module! {
             }
             Ok(())
         }
+
+        /// Freezes a given bridge transaction.
+        pub fn freeze_bridge_tx(origin, bridge_tx: BridgeTx<T::AccountId, T::Balance>) ->
+            DispatchResult
+        {
+            Self::is_root_or_admin(origin)?;
+            let proposal =
+                <T as Trait>::Proposal::from(Call::<T>::handle_bridge_tx(bridge_tx.clone())).into();
+            let proposal_id = <multisig::Module<T>>::proposal_ids(&Self::relayers(), &proposal);
+            ensure!(proposal_id.is_some(), Error::<T>::NoSuchProposal);
+            <FrozenTxs<T>>::insert(&bridge_tx, true);
+            Self::deposit_event(RawEvent::FrozenTx(bridge_tx));
+            Ok(())
+        }
+
+        /// Unfreezes a given bridge transaction.
+        pub fn unfreeze_bridge_tx(origin, bridge_tx: BridgeTx<T::AccountId, T::Balance>) ->
```

Comment:
Can we add a function that takes a vector of `BridgeTx` to unfreeze.

Reply:
Same for `freeze_bridge_tx` ;-)

Reply:
Sure, or maybe just convert this function to take a vector?

Reply:
Yep - that works.